### PR TITLE
add some minor suggestions

### PR DESCRIPTION
**Pull request by mellson**
Sorry about all the formatting changes, my default Prettier is eager to rearrange stuff 🙈

### CODE REVIEW

File: src/components/App/App.tsx [hunk 1]
```diff
@@ -58,8 +57,6 @@ const App = () => {
     },
   });
 
-  const [leversAreChanging, setLeversAreChanging] = useState(false);
```
Comment (mellson) [inline src/components/App/App.tsx:61]: Removing this to make the UI refresh when changing the levers

File: src/components/Controls/Controls.css.ts [hunk 2]
```diff
@@ -90,10 +90,10 @@ const flash = keyframes`
 `;
 
 export const FlashingControlButton = styled(ControlButton)<{
-  animate: boolean;
+  $animate: boolean;
```
Comment (mellson) [inline src/components/Controls/Controls.css.ts:93]: Using the ﻿$ symbol to change this to a transient prop to stop React from complaining about it

File: src/statechart/app.machine.ts [hunk 3]
```diff
@@ -39,6 +39,8 @@ export const appMachine =
       events: {} as AppMachineEvent,
     },
     tsTypes: {} as import('./app.machine.typegen').Typegen0,
+    predictableActionArguments: true,
```
Comment (mellson) [inline src/statechart/app.machine.ts:42]: Adding these to stop XState from complaining

**Review by kevinmaes — APPROVED** [review]
These changes are so helpful. Thanks a lot @mellson !

**Pull request merged**